go.mod

```diff
@@ -1,6 +1,15 @@
 module github.com/BurntSushi/go-sumtype
 
+go 1.17
```

Comment:
This was set to an older Go version as it seemed unnecessary to pull it all the way up to Go 1.19, and the new behavior of Go 1.19 is to add this line as `go 1.19` if it's unspecified. If you have opinions on setting it to something else, happy to update accordingly.

Reply:
I think 1.17 is "old enough" :-)